scripts/python/policia_nacional/README

```diff
@@ -0,0 +1,52 @@
+		 + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + +
```

Comment:
Normally the readme file is written with https://www.markdownguide.org/ (also add an extension .md)